website/addons/wiki/views.py

```diff
@@ -439,3 +443,73 @@ def project_wiki_validate_name(wname, **kwargs):
             message_long='A wiki page with that name already exists.'
         ))
     return {'message': wiki_name}
+
+@must_be_valid_project
+@must_be_contributor_or_public
+def project_wiki_grid_data(auth, wname, **kwargs):
+    node = kwargs['node'] or kwargs['project']
```

Comment:
This should no longer be necessary, I believe the `node` should always refer to the node being requested.